modules/running-distributed-data-science-workloads-disconnected-env.adoc

```diff
@@ -18,6 +18,18 @@ To run a distributed data science workload in a disconnected environment, you mu
 * You have logged in to {productname-long}.
 * You have created a data science project.
 
+ifndef::upstream[]
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, the project creator must provide you with Admin access, as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/configuring-access-to-data-science-projects_nb-server[Configuring access to data science projects].
+endif::[]
+ifdef::upstream[]
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, the project creator must provide you with Admin access, as described in link:{odhdocshome}/working-on-data-science-projects/#_configuring_access_to_data_science_projects[Configuring access to data science projects].
```

Comment:
Does it need to be the project creator? What if the project creator leaves the company or goes on vacation? Would it not be anybody with Admin access, or can only one user (the creator) have Admin access?

Reply:
Thinking about this again, if the project creator is unavailable, the RHOAI admin would be able to add a new project Admin for the project. That said, we still need to clarify if it needs to be the project creator that that must provide Admin access to the user, or can any Admin assigned to the project also provide this?

Reply:
Good catch, I overlooked that. Instead of "project creator" there should be "cluster admin".

Reply:
Project creator cannot change RBAC settings on OpenShift cluster, so it can't give others admin access.

Reply:
This section is talking about granting Admin access to the project, not to the cluster?

Reply:
Thank you @chtyler and @sutaakar - updated based on your feedback: )